testing/trino-tests/src/test/java/io/trino/tests/tpch/TestTpchLocalStats.java

```diff
@@ -52,6 +53,7 @@ public void setUp()
                 .setSystemProperty(PREFER_PARTIAL_AGGREGATION, "false")
                 // Stats for non-EXPLAIN queries are not collected by default
                 .setSystemProperty(COLLECT_PLAN_STATISTICS_FOR_ALL_QUERIES, "true")
+                .setSystemProperty(DEFAULT_FILTER_FACTOR_ENABLED, "false")
```

Comment:
Please enable it. Otherwise we need some rationale for running non-default config in CBO tests